server/server.js

```diff
@@ -28,6 +28,8 @@ app.use(bodyParser.json());
 //
 mongoose.connect(dbConfig.url);
 var connection = mongoose.connection;
+global.autoIncrement = require("mongoose-auto-increment");
```

Comment:
This and the declaration like it in the Grunt file is are necessary for the module to remain configured even through external files.